bears/general/RangeHelpers.py

```diff
@@ -134,6 +136,24 @@ def get_valid_sequences(file,
         if not sequence_line_text.rstrip().endswith(':') and check_ending:
             valid = False
 
+        if(keyword and
```

Comment:
maybe at a quick comment what exactly determines whether `valid` is `True` or `False`?